docs/sphinx/about/whats-new.txt

```diff
@@ -124,13 +148,21 @@ OME-XML changes include:
      maintenance and growth
    - allow for genuine abstract model types and enable C++ model
      implementation
+* updated OME-XML and OME-TIFF public sample files
 
 The Bio-Formats C++ native implementation has been decoupled from
 the Java codebase and will be released as
 `OME-Files C++ <http://downloads.openmicroscopy.org/ome-files-cpp/>`_ from now
 on, with the exception of OME-XML which is still within Bio-Formats at present
 (there is a plan to decouple both the Java and the C++ versions of OME-XML in
-future).
+future). All the C++ dependencies remaining in the main Bio-Formats repository
```

Comment:
The C++ part is incidental; this affects the Java code to the same extent.  Simply noting these components have been switched to 100% BSD licenced code would be sufficient.